pkg/hvsock/hvsock_windows.go

```diff
@@ -125,7 +126,7 @@ func bind(s syscall.Handle, a HypervAddr) error {
 }
 
 func accept(s syscall.Handle, a *HypervAddr) (syscall.Handle, error) {
-	return 0, errors.New("accept(): Unimplemented")
+	return sys_accept(s)
```

Comment:
This function is supposed to fill in the details for `a` as well (which the linux version does)